tools/topology/topology2/cavs-sdw-src-gain-mixin.conf

```diff
@@ -39,7 +39,7 @@ Define {
 #ALH Index: 0, Direction: duplex
 Object.Dai.ALH [
 	{
-		dai_index 2
```

Comment:
the commit message talks about pipeline ID and route ID but all im seeing is dai_index updates.

Reply:
> the commit message talks about pipeline ID and route ID but all im seeing is dai_index updates.

@ranj063 The pipeline ID and route ID is already grouped by f0a010052bd0. This commit follows the same rule for ALH dai index. I will update the commit message to make it more intuition.